### ISSUE TITLE
Incorrect error for parameter properties references

### ISSUE THREAD

**Issue by sandersn** [Microsoft/TypeScript#26719]
```ts
class Foo {
    constructor(public x = 12, public y = x.toFixed()) {}
}
```

**Expected behavior:**
No error.

**Actual behavior:**
Can't find name 'x' in the expression 'x.toFixed()'.

Caused by https://github.com/Microsoft/TypeScript/pull/26317, so the fix should be an easy modification to that new code.

**Comment by RyanCavanaugh** [comment]
Paging @Kingwl 😉

**Comment by Kingwl** [comment]
sorry about that, i will take a look soon
